SmartHomeManager/SmartHomeManager.API/Controllers/ProfileAPI/ProfilesController.cs

```diff
@@ -25,70 +25,82 @@ public ProfilesController(ProfileService profileService)
             _profileService = profileService ?? throw new ArgumentNullException("profile service null");
         }
 
-        /*// GET: api/Profiles
+        /* 
+         * GET: api/Profiles
+         * Return: 
+         * Ok(profiles) - IEnumerable of Profiles
+         * NotFound(1) - No Profiles in DB
+        */
         [HttpGet]
         public async Task<ActionResult<IEnumerable<Profile>>> GetProfiles()
         {
-            *//*if (_context.Profiles == null)
-            {
+            IEnumerable<Profile> profiles = await _profileService.GetProfiles();
+
+            if (profiles == null)
```

Comment:
this expression is never true because if there are no profiles, it returns you an empty IEnumerable instead of null. You might wanna check for capacity instead. Might wanna convert to list first though.